kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
+
 # Run the text based installer to see dependency info in screenshots
+text
```

Comment:
Do we need this change?  We do the GUI install as of 7eabb6206f286 because it's easier if we need to debug a failed build and VNC into the failed build.